2020-2021/Name, Blank/Calculator.py

```diff
@@ -0,0 +1,72 @@
+import math
+print("Keshav's calculator!")
```

Comment:
# Great work!
> Score: 8/10 (+ 7 for completion of the task and + 1 limited additions to functions and the user experience)
## Potential Improvements:
- Add more functions (esp. trig functions)
 - Work to get Order of Operations to work natively (code it or use parentheses)
- Add a GUI ;)
 ~ Ojasw Upadhyay